bin/node/runtime/src/lib.rs

```diff
@@ -541,7 +541,7 @@ parameter_types! {
 	pub const BondingDuration: sp_staking::EraIndex = 24 * 28;
 	pub const SlashDeferDuration: sp_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
-	pub const MaxNominatorRewardedPerValidator: u32 = 256;
+	pub const MaxNominatorRewardedPerPage: u32 = 256;
```

Comment:
I think I said this elsewhere as well, if we rename this, let's rename it to something shorter and better, e.g. `ExposurePageSize`, else let's keep it same.